docs/get-started/getting-started-alluxio.md

```diff
@@ -419,4 +419,93 @@ See the master.log and worker.log in this path.
 
 ### Auto start Alluxio the master and workers
 After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
-Systemd service can automatically start process if process is terminated.
\ No newline at end of file
+Systemd service can automatically start process if process is terminated.
+
+## Alluxio metrics
+### Monitor Alluxio metrics based on Alluxio Master Web
+Users can view the Alluxio metrics in the Web interface of Alluxio leading master:       
+http://<leading_master_host>:19999/metrics   
+For more details, please refer to section 3.1 of Alluxio doc: [Master Web UI Metrics](https://docs.alluxio.io/os/user/stable/en/operation/Metrics-System.html#default-http-json-sink)      
+The Alluxio Web UI is not available by default on databricks, the following provides a method to view the Web UI by SSH tunnel via jump server.   
+First forward the Alluxio port 19999 to a new port on a jump server, then forward the new port on the jump server to a local port, finally access the local port in the browser.   
+For example:   
+- Forward the port 29999 on jump server to local port 39999 on your own machine.   
+ssh -L 39999:localhost:29999 user@jump-server   
+- Forward the alluxio server 1999 port to the port 29999 on jump server.   
```

Comment:
Alluxio should be capitalized.
I assume this is 19999 port?

Reply:
The forward paths are:

Alluxio-master: 19999 -> jump-server:29999 -> local-pc:39999

ssh -L 39999:localhost:29999 user@jump-server
The above command means forward port 29999 on jump-server to the local 39999 port.

Reply:
Done